core/src/main/java/org/apache/carbondata/core/util/DataLoadMetrics.java

```diff
@@ -21,10 +21,10 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class OutputFilesInfoHolder implements Serializable {
-
-  private static final long serialVersionUID = -1401375818456585241L;
-
+/**
+ * store data loading metrics
+ */
+public class DataLoadMetrics implements Serializable {
```

Comment:
I didn't call it metrics initially because it has fileNames, partition path and all. 
You think metrics is more suitable ?

Reply:
yes. For Hadoop framework, we collect them and put them to the task message; For Spark framework, we collect them and put them to the task metrics.